i18n/zh/docusaurus-plugin-content-docs/current/architecture/edge/devicetwin.md

```diff
@@ -2,353 +2,327 @@
+- **查询设备**: 从设备表中查询设备
 
-- **Query Device All**: Displays all the devices present in the device table
+- **查询全部设备**: 显示设备表中存在的所有设备
 
-- **Update Device Multi**: Updates multiple columns of multiple devices in the device table
+- **Update Device Multi**: 更新设备表中多个设备的多列
 
-- **Add Device Trans**: Inserts device, device attribute and device twin in a single transaction, if any of these operations fail,
-   then it rolls back the other insertions
+- **Add Device Trans**: 一次性插入设备、设备属性和设备孪生，如果其中任何操作失败，则会回滚其他插入
 
-- **Delete Device Trans**: Deletes device, device attribute and device twin in a single transaction, if any of these operations fail,
-   then it rolls back the other deletions
+- **Delete Device Trans**: 一次中删除设备、设备属性和设备孪生，如果其中任何操作失败，则会回滚其他删除操作
 
 
-### Device Attribute Table
+### 设备属性表
 
-Device attribute table contains the data regarding the device attributes associated with a particular device in the edge node.
-The following are the columns present in the device attribute table :
+设备属性表包含有关与边缘节点中的特定设备关联的设备属性的数据。以下是设备属性表中存在的列：
 
-| Column Name | Description |
+| 列名 | 描述 |
 |----------------|--------------------------|
-| **ID** |  This field indicates the id assigned to the device attribute |
-| **DeviceID** |  This field indicates the device id of the device associated with this attribute |
-| **Name** | This field indicates the name of the device attribute |
-| **Description** | This field indicates the description of the device attribute |
-| **Value** | This field indicates the value of the device attribute |
-| **Optional** | This fields indicates whether the device attribute is optional or not |
-| **AttrType** | This fields indicates the type of attribute that is referred to |
-| **Metadata** |This fields describes the metadata associated with the device attribute  |
+| **ID** |  该字段表示分配给设备属性的id |
+| **DeviceID** |  该字段表示与该属性关联的设备的设备ID |
+| **Name** | 该字段表示设备属性的名称 |
+| **Description** | 该字段表示设备属性的描述 |
+| **Value** | 该字段表示设备属性的值 |
+| **Optional** | 该字段指示设备属性是否可选 |
+| **AttrType** | 该字段指示所引用的属性的类型 |
+| **Metadata** |该字段描述与设备属性关联的元数据  |
 
 
-**Operations Performed:**
+**可执行的操作**
 
-The following are the operations that can be performed on this data :
+以下是可以对此数据表执行的操作：
 
-   - **Save Device Attr**: Inserts a device attribute in the device attribute table
+   - **Save Device Attr**: 在设备属性表中插入设备属性
 
-   - **Delete Device Attr By ID**: Deletes a device attribute by its ID from the device attribute table
+   - **Delete Device Attr By ID**: 从设备属性表中按 ID 删除设备属性
 
-   - **Delete Device Attr**: Deletes a device attribute from the device attribute table by filtering based on device id and device name
+   - **Delete Device Attr**: 通过根据设备 id 和设备名称进行过滤，从设备属性表中删除设备属性
 
-   - **Update Device Attr Field**: Updates a single field in the device attribute table
+   - **Update Device Attr Field**: 更新设备属性表中的单个字段
 
-   - **Update Device Attr Fields**: Updates multiple fields in the device attribute table
+   - **Update Device Attr Fields**: 更新设备属性表中的多个字段
```

Comment:
The original text has issues,   should be `Update Device Attr  Multi Fields`

perhaps it can be modified together.